src/pages/DualIndexPlate.tsx

```diff
@@ -163,6 +172,20 @@ function DualIndexPlate() {
               </div>
             </div>
           </div>
+          {sourceReagentPlate && (
+            <div className="w-1/4 mt-4 mb-4" id="plateType">
+              <Label name={'Plate Type'}>
+                <Select emptyOption onChange={(e) => handlePlateTypeChange(e.currentTarget.value)}>
+                  {PLATE_TYPES.map((plateType, index) => (
+                    <option key={index} value={plateType}>
```

Comment:
if plateType is unique you may as well use it as the key as well, think index should only be used as last resort key.

Reply:
Yes, Added by accident. plateType is the right field